.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           cache-dependency-path: "pyproject.toml"
       - name: Run static analysis and format checkers
         run: |
-          pip install .[testing,transforms] tox
+          pip install .[testing,transforms] tox-uv tox
```

Comment:
@jonas-w We could try to replace this pip install with uv pip install, but that means we would need to setup uv before that. Need to research what's the quickest way to do this.